src/auto-evo/simulation/food_source/RandomFoodSource.cs

```diff
@@ -0,0 +1,17 @@
+namespace AutoEvo
+{
+    using System;
+
+    /// <summary>
+    ///   A class for random-encounter based food sources.
+    /// </summary>
+    public abstract class RandomFoodSource : FoodSource
```

Comment:
Why was this renamed? I think the name RandomFoodSource is not any better than CompoundFoodSource for finding compounds.

Reply:
It was not renamed. CompoundFoodSource still exists. Here, randomFoodSource includes chunks, other species...

 It is just that I separated Environmental Food sources (whose gathering is not chance-based) from the rest, whose gathering is encounter-based.

Reply:
Ah, I reread the code and realized that. Still I think random is super non-descriptive name, I'd assume it would actually use just random noise as the results... Something with "encounter" in the name would be more clear. Or at least not as easily confused with random numbers.